src/components/Menu/components/MenuGroup/MenuGroup.tsx

```diff
@@ -55,8 +55,8 @@ export default class MenuGroup extends React.Component<Props, never> {
   }
 
   private handleClose = () => {
-    const {title, onClose} = this.props;
-    onClose(title);
+    const {onClose} = this.props;
+    onClose();
```

Comment:
This was no longer necessary as `onClose` wouldn't be called with a `title` that was not active.